src/auth/tenant.ts

```diff
@@ -71,6 +112,7 @@ export class Tenant {
    * @param {TenantOptions} tenantOptions The properties to convert to a server request.
    * @param {boolean} createRequest Whether this is a create request.
    * @return {object} The equivalent server request.
+   * @internal
```

Comment:
I wasn't sure about the best approach for these two methods. They're not supposed to be publicly exposed, but if I pulled them out it might eliminate the logical grouping it has with `Tenant`. I also considered doing an interface/implementation split here. Any thoughts?

Reply:
Looking at their functionality, I think it's ok to pull them from this class. We can put them in `tenant-internal` as top-level methods or as static methods of a new `TenantUtils` class.